src/world/area_kmr/kmr_03/8C8140.c

```diff
@@ -15,60 +15,38 @@ ApiStatus func_802401B0_8C8140(ScriptInstance* script, s32 isInitialCall) {
 #include "world/common/UnkPositionFunc.inc.c"
 
 // 8C8680
-Script M(script_ExitWalk_802406F0) = {
-    SI_GROUP(27),
-    SI_CALL(0x802D216C, 60, 0),
-    SI_EXEC(0x80285CF4),
-    SI_CALL(0x802CA400, "kmr_04", 0),
-    SI_WAIT_FRAMES(100),
-    SI_RETURN(),
-    SI_END(),
-};
+Script M(ExitWalk_802406F0) = EXIT_WALK_SCRIPT(60, 0, "kmr_04", 0);
```

Comment:
does disasm_map produce these now?

Reply:
No, we've had this macro for a while and I just noticed it